assets/settings/default.json

```diff
@@ -95,7 +95,10 @@
     "left_padding": 0.2,
     // The relative width of the right padding of the central pane from the
     // workspace when the centered layout is used.
-    "right_padding": 0.2
+    "right_padding": 0.2,
+    // Whether to show the borders of the padding areas when the centered
+    // layout is used.
+    "show_border": true
```

Comment:
I wonder, how useful would it be to make border more configurable with `"border_size": 0.0,` instead?
Then, both groups who do not want the border and who want it thicker will be good.